### PR TITLE
Add absolute value, aliquot sum, ceil, and square root

### PR DESCRIPTION
**Pull request by snood1205**
Add math algorithms including absolute value, aliquot sum, ceil, and square root. Square root was calculated using Newton's method. All of these have test coverage as well.

### CODE REVIEW

**Review by vbrazo — COMMENTED** [review]
That's awesome man. Could you also update the DIRECTORY.md?

**Comment by snood1205** [comment]
> That's awesome man. Could you also update the DIRECTORY.md?

Absolutely, sorry for the delay, getting back to work this week has had me quite busy, but I'll update that today or tomorrow so we can close this out.

File: maths/square_root_test.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'minitest/autorun'
+require_relative './square_root'
+
+class SquareRootTest < Minitest::Test
+  def test_negative_number
+    assert_raises { SquareRoot.call(-1) }
```
Comment (vbrazo) [inline maths/square_root_test.rb:8]:
```suggestion
    assert_raises DomainError do
      SquareRoot.call(-1)
    end
```

**Review by vbrazo — APPROVED** [review]
Looks good, just left a minor comment that asks you to specify which Error class you are asserting in the test. Thanks!

**Pull request merged**